unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs

```diff
@@ -467,17 +481,68 @@ internal bool NeedToRefreshRealms(IEnumerable<RealmModel> newRealmList)
 
     internal void UpdateProfileInfo(UserProfile profile)
     {
+        view.currentProfileCard.SetIsClaimedName(profile.hasClaimedName);
         view.currentProfileCard.SetProfileName(profile.userName);
         view.currentProfileCard.SetProfileAddress(profile.ethAddress);
         view.currentProfileCard.SetProfilePicture(profile.face128SnapshotURL);
     }
 
     internal void OnCloseButtonPressed(bool fromShortcut)
     {
+        if (DataStore.i.exploreV2.isSomeModalOpen.Get())
+            return;
+
         if (isOpen.Get())
             exploreV2Analytics.SendStartMenuVisibility(false, fromShortcut ? ExploreUIVisibilityMethod.FromShortcut : ExploreUIVisibilityMethod.FromClick);
 
-        SetVisibility(false);
+        if (!fromShortcut)
+            SetVisibility(false);
+        else
+        {
+            if (isOpen.Get())
+                SetVisibility(false);
```

Comment:
since the `else` has brackets, can we add brackets here as well?

Reply:
I love your TOC ❤️ Thanks! Done in https://github.com/decentraland/unity-renderer/pull/1766/commits/2d28daf31911a106f173fd5a1b48b38783649ce1